### ISSUE TITLE
Adding "We Work Remotely"?

### ISSUE THREAD

**Issue by ColeTownsend** [johnmartin/remote#2]
This is a remote jobs board. All sorts of stuff. Relevant to this would be https://weworkremotely.com/categories/1/jobs and https://weworkremotely.com/categories/2/jobs.

I tried added the design jobs locally:

``` json
{
    "site": "We Work Remotely",
    "base_url": "https://weworkremotely.com",
    "url": "https://weworkremotely.com/categories/1/jobs",
    "rules": {
      "entries": "#category-1 article li",
      "url": "> a ",
      "title": "{entry} .title",
      "company": "{entry} .company",
      "type": false,
      "where": false,
      "description": false
    }
  },
```

Unfortunately nothing comes up for design anywhere, which is surprising.

**Comment by johnmartin** [comment]
@ColeTownsend Ahoy, I've taken a look and it was because of some buggy logic in `./server/update.js` (I fixed it in 3c448d1). Give yourself a `git pull` & `meteor` and everything should be tickety boo.

**Comment by ColeTownsend** [comment]
You're an all-star. Thanks a ton!
